### PR TITLE
modification of the schema.json 

### PR DESCRIPTION
**Pull request by Helarga**
The system parameter schema.json needed to be reviewed and updated to reflect the latest modelica models.

The PR addresses issue https://github.com/urbanopt/geojson-modelica-translator/issues/240 where 

- Buildings parameters have been updated to include T heating/cooling air set_point, Chilled/heating water supply and return temperature for building and district level.

- Removed  a non required old parameter section" central plant".

### CODE REVIEW

File: geojson_modelica_translator/system_parameters/schema.json [hunk 1]
```diff
@@ -287,91 +269,137 @@
...
-          "default": 999
+          "default": 500
         },
-        "cooling_supply_water_temperature_district": {
-          "description": "Cooling supply water temperature for district (deg C)",
+        "TDis_ChWS": {
+          "description": "Chilled water supply temperature district side (K)",
           "type": "number",
-          "default": 5
+          "default": 5+273.15
```
Comment (vtnate) [inline geojson_modelica_translator/system_parameters/schema.json:313]: I don't think json allows the `+` symbol in a value. May need to make these default values just a number. For instance, this one would be `278.15`.

File: geojson_modelica_translator/system_parameters/schema.json [hunk 2]
```diff
@@ -199,7 +199,7 @@
           "enum": [
             "None",
             "Indirect Heating and Cooling",
-            "Heat Exchanger Only"
+            "Direct Heating and Cooling"
```
Comment (vtnate) [inline geojson_modelica_translator/system_parameters/schema.json:202]: Python code will have to be changed to match these new values you're setting. I can work on that if that will be helpful to you.

File: geojson_modelica_translator/system_parameters/schema.json (hunk 1 above)
Comment (Helarga) [reply geojson_modelica_translator/system_parameters/schema.json:313]: ok i will fix it

File: geojson_modelica_translator/system_parameters/schema.json (hunk 2 above)
Comment (Helarga) [reply geojson_modelica_translator/system_parameters/schema.json:202]: the "Heat Exchanger only" type is a mistake actually, i corrected it by "Direct Heating and Cooling"

**Comment by nllong** [comment]
Move to urbanopt organization... track here: https://github.com/urbanopt/geojson-modelica-translator/pull/274

**Pull request closed without merging**